src/index.html

```diff
@@ -6,7 +6,44 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
-  </body>
+  <div class="calendar">
+    <div class="calendar__start-day-wed"></div>
+    <div class="calendar__month-length-29"></div>
+
+    <div class="calendar__sheet" >
```

Comment:
```suggestion
  <div class="calendar calendar--start-day--wed calendar--month-length--29">
```
Modifiers should be added to the block directly, they are not its elements
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/4292c4c4-030c-4b2a-84e6-3998daf675e4)